Plugins/NativeWindow/Source/NativeWindow.cpp

```diff
@@ -37,8 +37,13 @@ namespace Babylon::Plugins::Internal
     {
     }
 
-    void NativeWindow::Resize(size_t newWidth, size_t newHeight)
+    void NativeWindow::Resize(size_t newWidth, size_t newHeight, void* newWindowPtr)
```

Comment:
Nit: Can we match parameter order with `NativeEngine::Reinitialize` for consistency?
```suggestion
    void NativeWindow::Resize(void* newWindowPtr, size_t newWidth, size_t newHeight)
```

Reply:
My first thought was to add a setWindowPtr and let the resize with just width,height. newWindowPtr is used on iOS only. I don't like other platform to be forced to call the function with a nullptr. What do you think @syntheticmagus ?